jaxrs-spec/src/main/asciidoc/chapters/providers/_entity_providers.adoc

```diff
@@ -128,12 +128,12 @@ type combinations:
   All media types (`/*`).
 `java.io.File`::
   All media types (`/*`).
-`javax.activation.DataSource`::
+`jakarta.activation.DataSource`::
```

Comment:
I'm fine with already updating this reference, because it looks like the platform project decided to move the Activation API to the jakarta namespace as well. Same for JAXB.

Reply:
I don't like the idea to have 404's in the preview. Please let's reflect the real truth in `master` always. Let's keep this PR "on hold" until there is at least a preview availabile on that url.

Reply:
Just to clarify. With "404" you mean that we reference classes that don't exist yet?

Reply:
Yes. :-)